### PR TITLE
Create get account info api

### PR DESCRIPTION
**Pull request by Arturnix**
- Create class to manage program flow: CharacterViewerManager ;
- Create class to GET account info using API: GetApiAccount ;
- Create menu in main;

### CODE REVIEW

File: src/main/java/com/diablo3CharViewer/FetchToken.java
Comment (blazejprzyluski) [inline src/main/java/com/diablo3CharViewer/FetchToken.java:1]: Please include review changes from the first PR

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package com.diablo3CharViewer;
+
+public class GetApiAccount {
+
+    private String region = "eu";
```
Comment (blazejprzyluski) [inline src/main/java/com/diablo3CharViewer/GetApiAccount.java:5]: make final

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    private String locale = "pl_PL";
```
Comment (blazejprzyluski) [inline src/main/java/com/diablo3CharViewer/GetApiAccount.java:6]: make final

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    private String account;
+
+    private String requestURL;
+
+    //mozna ustawic dla innych regionow
```
Comment (blazejprzyluski) [inline src/main/java/com/diablo3CharViewer/GetApiAccount.java:11]: Do you want to use different regions in your app?

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        return account;
+    }
+
+    public String getRequestURL() {
+        return requestURL;
+    }
+
+    //zrobic jako static?
+    public String generateRequest(FetchToken fetchToken) { //token przypisywać ze zmiennej. Pobieram token kiedy program startuje i wrzucam go do stałej.
+
```
Comment (blazejprzyluski) [inline src/main/java/com/diablo3CharViewer/GetApiAccount.java:47]: unnecessary white space

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    public String getAccount() {
+        return account;
+    }
+
+    public String getRequestURL() {
+        return requestURL;
+    }
+
+    //zrobic jako static?
```
Comment (blazejprzyluski) [inline src/main/java/com/diablo3CharViewer/GetApiAccount.java:45]: could be a static method, yes, but outside of this class

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java (hunk 1 above)
Comment (Arturnix) [reply src/main/java/com/diablo3CharViewer/GetApiAccount.java:5]: field deleted

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java (hunk 2 above)
Comment (Arturnix) [reply src/main/java/com/diablo3CharViewer/GetApiAccount.java:6]: field deleted

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java (hunk 3 above)
Comment (Arturnix) [reply src/main/java/com/diablo3CharViewer/GetApiAccount.java:11]: only eu region

File: src/main/java/com/diablo3CharViewer/GetApiAccount.java (hunk 5 above)
Comment (Arturnix) [reply src/main/java/com/diablo3CharViewer/GetApiAccount.java:45]: now there is only 1 method left in this class. Move this method to manager class as its method and remove class to handle account api?

**Pull request merged**